aflr/api_resources/file.py

```diff
@@ -1,19 +1,62 @@
+from posixpath import basename
 import aflr
-from aflr.api_request import APIRequest
+from aflr.helper_classes import ListableResource
+from tkinter import Tk
+from tkinter.filedialog import askopenfilename
```

Comment:
I wouldn't add this package. Instead we can have an argument for `file_path`. If users want to visually choose a file, they can use tkinter from their scripts. this means we also don't need basename

you can handle this with `open(path, "rb")`

Reply:
done